{"name": "test_login_params[data_for_login2]", "status": "passed", "attachments": [{"name": "log", "source": "f3fa4947-471d-4e25-b7af-1c6d428afb65-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data_for_login", "value": "('adelukesh@gmail.com', '48465', 'Fail')"}], "start": 1749456711091, "stop": 1749456718059, "uuid": "620a50ce-e985-452c-8939-00399cdd20e4", "historyId": "17f2997ed10c10d62350784f3bbf7451", "testCaseId": "5fe6fe3249a01c883d8efece63f2c386", "fullName": "test_login_params.TestLoginParams#test_login_params", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "regression"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "suite", "value": "test_login_params"}, {"name": "subSuite", "value": "TestLoginParams"}, {"name": "host", "value": "lukesh"}, {"name": "thread", "value": "22996-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login_params"}]}